typescript-primitives-and-variables/main.ts

```diff
@@ -0,0 +1,18 @@
+const fullName: string = 'Katelyn Ngo';
+const isCool: boolean = true;
+const totalPets: number = 1;
+
+console.log(' Value of Full Name:', fullName);
+console.log('Value of isCool:', isCool);
+console.log('Value of totalPets:', totalPets);
+console.log('typeof fullName:', typeof fullName);
+console.log('type of isCool:', typeof true);
+console.log('typeof totalPets:', typeof 1);
+
+const empty: null = null;
+console.log('value of empty:', empty);
+console.log('typeof empty:', typeof empty);
+
+let nothing: undefined;
+console.log('Value of nothing:', nothing);
+console.log('type of nothing:', nothing);
```

Comment:
It should be the typeof nothing here, otherwise you are logging the same value as line 17